includes/cleanup.php

```diff
@@ -68,3 +104,19 @@ function pmpro_delete_post( $post_id = null ) {
 	);
 }
 add_action( 'delete_post', 'pmpro_delete_post' );
+
+/**
+ * Delete all membership data for a specific user from the membership users table.
+ *
+ * @param int $user_id The WordPress user ID.
+ * @since TBD
+ */
+function pmpro_delete_membership_history( $user_id ) {
+	global $wpdb;
```

Comment:
We should bail here (return, maybe return false) if $user_id is empty. Otherwise, I think our query here deletes all user history.

Reply:
Added return false here as well to match the above check as well. That way if the function returns false we know the user ID is empty.